src/AngryBytes/Hash/HasherInterface.php

```diff
@@ -31,4 +31,14 @@
      * @return string
      **/
     public function hash($data, $salt);
```

Comment:
Ik denk het niet, voor de md5 hasher is de salt verplicht maar voor de password hasher weer niet.